docs/slides/slides-introduction.md

```diff
@@ -0,0 +1,246 @@
+---
+# try also 'default' to start simple
+theme: default
+# random image from a curated Unsplash collection by Anthony
+# like them? see https://unsplash.com/collections/94734566/slidev
+#background: https://cover.sli.dev
+background: /img/race.jpg
+# some information about your slides (markdown enabled)
+title: Welcome to Metaxy
+info: |
+  ## Introducing Metaxy
+  Rapid feature exploration unlocked - at a budget.
+
+  Learn more at [Metaxy](https://docs.metaxy.io)
+# apply UnoCSS classes to the current slide
+class: text-center
+# https://sli.dev/features/drawing
+drawings:
+  persist: false
+# slide transition: https://sli.dev/guide/animations.html#slide-transitions
+transition: slide-left
+# enable MDC Syntax: https://sli.dev/features/mdc
+mdc: true
+# duration of the presentation
+duration: 20min
+routerMode: hash
+---
+
+# Welcome to Metaxy
+
+Accelerating AI experimentation with smart metadata handling without massive cost.
```

Comment:
motivate why - change of compute paradigma: cost of GPU compute, rapid experimentation, reproducibility